lib/phoenix_live_component.ex

```diff
@@ -81,6 +82,13 @@ defmodule Phoenix.LiveComponent do
   for stateful components. LiveView will emit warnings in future versions if
   this is not the case.
 
+  The assigns given to [`live_component/3`](`Phoenix.LiveView.Helpers.live_component/3`)
+  are handled as for a stateless component: either passed to
+  `c:update/2` if it is defined; or, merged to the socket assigns
+  otherwise.  The optional `c:preload/1` receives a list of the assigns
+  maps for all instances of the component in the parent LiveView, see
+  below for an example of how this may be helpful.
```

Comment:
I think the order here is backwards.  We are talking about update and preload before we introduce mount and so on. Maybe we should move it later a bit? Also perhaps we should clearly define the workflow without referencing stateless components.

Reply:
Yes, I agree the assigns should move before discussion of state because they are common to both.  I considered it but didn't want to introduce too many changes here. :)  I'll do that and update soon.

Reply:
I think it is fine to repeat yourself in the docs. :)